### DISCUSSION TITLE
Parameter loading for a particular model?

### DISCUSSION THREAD

**Discussion by CalculonPrime** [oobabooga/text-generation-webui#4150] [Q&A] [answered]
Can the "Generation" and "Instruction template" settings be associated with a particular model so that they are loaded when you select that model?  Seems like each model on huggingface lists out a preferred instruction template so it would be natural to at least associate that.

Or, are the parameters always loaded and saved completely separately from the current model?

**Comment by missionfloyd** [comment, marked as answer]
After saving model settings, you can add additional settings to `models/config-user.yaml`.
https://github.com/oobabooga/text-generation-webui/blob/0197fdddf106270ccf77fc2757d7bc53cc9729d4/models/config.yaml#L24
https://github.com/oobabooga/text-generation-webui/blob/0197fdddf106270ccf77fc2757d7bc53cc9729d4/settings-template.yaml#L8
